.github/workflows/runtests.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build and Test (Unit, Lint, E2E)
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build_and_unit_test:
+  #Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
I would remove version 10 and 15, since 10 is EOL at the end of April, and 15 isn't an LTS release.

Reply:
:)  alright